content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -180,53 +180,57 @@ Make sure the default namespace does not include the label `istio-injection=enab
 
 1. Deploy an ingress gateway so you can access the bookinfo app from outside the cluster:
 
-{{< tabset category-name="config-api" >}}
+    {{< warning >}}
+    To get IP address assignment for `Loadbalancer` service types in `kind`, you may need to install a tool like [MetalLB](https://metallb.universe.tf/). Please consult [this guide](https://kind.sigs.k8s.io/docs/user/loadbalancer/) for more information.
+    {{</ warning >}}
 
-{{< tab name="Istio APIs" category-value="istio-apis" >}}
+    {{< tabset category-name="config-api" >}}
```

Comment:
This tabset should not be indented (you should instead use `3)` for the following list entry instead, like it was previously. The problem is that `{{< text }}` entries nested within a tabset don't format properly. If they have specific indentation, it gets lost. Look at the `sed` ccommand, below, in the preview to see an example (the problem is even worse if the text section includes yaml). The good news is that text inside a tabset displays somewhat indented anyway, so the list ends up looking fine even if the actual tabeset is not indented.

Reply:
ah gotcha, we were discussing why it's this way, will revert!